docs/wiki/PipelineRemovalAction.md

```diff
@@ -0,0 +1,67 @@
+# Removal action
+
+This section describes how the removal of resources deployed by a module is performed and how to modify the default behaviour if a specific module or resource type needs it.
+
+---
+
+### _Navigation_
+
+- [Overview](#Overview)
+- [How it works](#how-it-works)
+- [Create a specialized removal procedure](#create-a-specialized-removal-procedure)
+
+---
+
+## Overview
```

Comment:
```suggestion
# Overview
```

Reply:
For consistency

Reply:
I will get this change for the sake of alignment between the different pages, but having multiple 1st level title is quite ugly, especially if we have a clear hierarchical relationship between sections (e.g. the "overview" of the "removal action")

Reply:
I do agree, BUT, let me at least lay down why:
- GitHub unfortunately does not support page headers - all has to be in text
- The title of the page naturally has to be `#` as otherwise it will not show up prominently as the page title
- All subsequent headers could be `##` (and that's how it was initially, actually). However, this essentially wasts one `#` and this becomes a problem for pages that use e.g. 4 indents as that's when it stops. This became a problem for several wiki pages, so we refactored it to essentially have the page title on the same level as the actual main chapter headers